apps/frontend/src/emails/templates/moderation/ReportStatusUpdated.vue

```diff
@@ -0,0 +1,44 @@
+<script setup lang="ts">
+import { Heading, Link as VLink, Section, Text } from '@vue-email/components'
+
+import StyledEmail from '@/emails/shared/StyledEmail.vue'
+</script>
+
+<template>
+	<StyledEmail
+		title="Report status updated"
+		:manual-links="[
+			{ link: 'https://modrinth.com/legal/rules', label: 'Community Guidelines' },
+			{
+				link: 'https://modrinth.com/dashboard/report/{report.id}',
+				label: 'View report in dashboard',
+			},
+			{ link: 'https://support.modrinth.com', label: 'Support Portal' },
+		]"
+	>
+		<Heading as="h1" class="mb-2 text-2xl font-bold">Report status updated</Heading>
+
+		<Text class="text-base">Hi {user.name},</Text>
+
+		<Text class="text-base">Your report, {report.id}, from {report.date} has been updated.</Text>
+
+		<Section class="bg-bg-super mb-4 mt-4 rounded-lg border border-divider pb-4 pl-4 pr-4 pt-4">
+			<Text class="m-0 text-base">{report.update}</Text>
```

Comment:
what is report.update? is it the body of a message? if so, that's markdown and i'm not sure this is rendering that? otherwise i'm struggling to figure out what this means and it seems kinda randomly placed here with no heading or title or explanation in the prior sentence on what is below